examples/website-demos/social-messaging/src/assets/LeftReply.tsx

```diff
@@ -0,0 +1,5 @@
+export const LeftReply: React.FC = () => (
+  <svg width="17" height="33" viewBox="0 0 17 33" fill="none" xmlns="http://www.w3.org/2000/svg" style={{ position: 'relative', bottom: '6px' }}>
+    <path d="M0.5 32V32C9.33656 32 16.5 24.8366 16.5 16L16.5 0" stroke="#DBDDE1"/>
```

Comment:
do the colors on these reply icons need to be variables?

Reply:
Oh yeah good catch